Traceback (most recent call last):
  File "/media/SSD4/dvegaa/DELFOS/DELFOS/img_script/main_img.py", line 30, in <module>
    wandb.log({"args": args,
  File "/home/dvegaa/anaconda3/envs/medvit/lib/python3.11/site-packages/wandb/sdk/wandb_run.py", line 440, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dvegaa/anaconda3/envs/medvit/lib/python3.11/site-packages/wandb/sdk/wandb_run.py", line 392, in wrapper_fn
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dvegaa/anaconda3/envs/medvit/lib/python3.11/site-packages/wandb/sdk/wandb_run.py", line 382, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dvegaa/anaconda3/envs/medvit/lib/python3.11/site-packages/wandb/sdk/wandb_run.py", line 1861, in log
    self._log(data=data, step=step, commit=commit)
  File "/home/dvegaa/anaconda3/envs/medvit/lib/python3.11/site-packages/wandb/sdk/wandb_run.py", line 1574, in _log
    self._partial_history_callback(data, step, commit)
  File "/home/dvegaa/anaconda3/envs/medvit/lib/python3.11/site-packages/wandb/sdk/wandb_run.py", line 1405, in _partial_history_callback
    self._backend.interface.publish_partial_history(
  File "/home/dvegaa/anaconda3/envs/medvit/lib/python3.11/site-packages/wandb/sdk/interface/interface.py", line 670, in publish_partial_history
    item.value_json = json_dumps_safer_history(v)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dvegaa/anaconda3/envs/medvit/lib/python3.11/site-packages/wandb/util.py", line 814, in json_dumps_safer_history
    return dumps(obj, cls=WandBHistoryJSONEncoder, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dvegaa/anaconda3/envs/medvit/lib/python3.11/json/__init__.py", line 238, in dumps
    **kw).encode(obj)
          ^^^^^^^^^^^
  File "/home/dvegaa/anaconda3/envs/medvit/lib/python3.11/json/encoder.py", line 200, in encode
    chunks = self.iterencode(o, _one_shot=True)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dvegaa/anaconda3/envs/medvit/lib/python3.11/json/encoder.py", line 258, in iterencode
    return _iterencode(o, 0)
           ^^^^^^^^^^^^^^^^^
  File "/home/dvegaa/anaconda3/envs/medvit/lib/python3.11/site-packages/wandb/util.py", line 779, in default
    return json.JSONEncoder.default(self, obj)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dvegaa/anaconda3/envs/medvit/lib/python3.11/json/encoder.py", line 180, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type Namespace is not JSON serializable
